apps/meteor/tests/e2e/page-objects/omnichannel-livechat.ts

```diff
@@ -88,6 +88,18 @@ export class OmnichannelLiveChat {
 		await this.btnNewChat.click();
 	}
 
+	async openAnyLiveChatAndSendMessage(
+		liveChatUser: { name: string; email: string },
+		message: string,
+		isOffline = true,
+		department?: string,
```

Comment:
It would be better to have an object as parameter here, this way we avoid to provide "magic" booleans to the function, it will be a named parameter instead, thus more meaningful

Reply:
oh, nice, thank you! I added it there, can you check again?